shablona/shablona.py

```diff
@@ -3,7 +3,15 @@
 from matplotlib import mlab 
 from scipy.special import erf
 import scipy.optimize as opt
+import os.path as op
 
+def get_data_path(absolute = True):
+    """Use special attribute __file__ to return the path to the module data.
```

Comment:
We might want to use the numpy docstring standard here (`Parameters`, `Returns`, etc): 

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt